```
pandas/io/json/_table_schema.py:16: error: Module "pandas._libs.json" has no attribute "ujson_loads"  [attr-defined]
pandas/io/json/_json.py:24: error: Module "pandas._libs.json" has no attribute "ujson_dumps"  [attr-defined]
pandas/io/json/_json.py:24: error: Module "pandas._libs.json" has no attribute "ujson_loads"  [attr-defined]
pandas/io/excel/_xlsxwriter.py:264: error: Module has no attribute "ujson_dumps"  [attr-defined]
pandas/io/excel/_odswriter.py:250: error: Module has no attribute "ujson_dumps"  [attr-defined]
```

--- a/pandas/_libs/json.pyi
+++ b/pandas/_libs/json.pyi
@@ -3,7 +3,7 @@ from typing import (
     Callable,
 )
 
-def dumps(
+def ujson_dumps(
     obj: Any,
     ensure_ascii: bool = ...,
     double_precision: int = ...,
@@ -14,7 +14,7 @@ def dumps(
     default_handler: None
     | Callable[[Any], str | float | bool | list | dict | None] = ...,
 ) -> str: ...
-def loads(
+def ujson_loads(
     s: str,
     precise_float: bool = ...,
     numpy: bool = ...,

